### PR TITLE
Set up a Hello Microverse project

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
this branch modifies the project, by adding the following features:

- html and css linter to the project.
- .gitignore file to the project.
- a header to the web page.
- style to the header.

### CODE REVIEW

**Review by KDlamini — COMMENTED** [review]
## Status: Invalid 🔴 

### Hi @Mhamad-Raad 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that does not have a PR description.

- Kindly write a summary of what you did in this feature in the PR description.
- Kindly write a meaningful PR title something similar to `Setup a hello Microverse project` instead of 'feature'.
- In the future, kindly give the branch a name that is relevant to the feature. In this case, your branch name could have been something like `setup` or `project-setup`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Mhamad-Raad** [comment]
Hi there

I am terribly sorry, it was a technical error from my side, I have already
contacted the success team hopefully I can get ine of review back.

Best regards
Mhamad Raad

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
```
Comment (MeqdamAlqudah) [inline .github/workflows/linters.yml:3]: - [ ]  Please commits should be in the present tense like add linter, add h1 tag, or something like this
image

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+env:
+  FORCE_COLOR: 1
```
Comment (MeqdamAlqudah) [inline .github/workflows/linters.yml:6]: - [ ] Please make sure to write a good description in your pull request telling exactly what you have done in this pull request
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Mhamad-Raad, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1:  Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Mhamad-Raad) [reply .github/workflows/linters.yml:3]: Alright, do you want me to add new commits or what to do to resolve the issue ?

**Comment by KDlamini** [comment]
> Hi there I am terribly sorry, it was a technical error from my side, I have already contacted the success team hopefully I can get ine of review back. Best regards Mhamad Raad

Hi @Mhamad-Raad 

No worries, all you need to do is edit your PR description and write a summary explaining the features you added to this project.
It's also possible to edit your PR title as well.

- There's nothing you have to do with the commit messages, but you can make sure that you improve the way you write your commit messages in the future.

Cheers,
Simo

**Review by codecaiine — APPROVED** [review]
Hi @Mhamad-Raad , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linters errors :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Optional suggestions

- [x] [OPTIONAL] Please edit your Pull Request to make it a more descriptive t, it should tell what changes are inside to the reviewer. A good example will be **Set up a Hello Microverse project** instead of  **First project feature**  :+1: 
- [ ] [OPTIONAL] I would recommend you to give a descriptive name to your branch that should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
- [ ] [OPTIONAL]  Please edit also your Pull Request summary because like your PR title the summary should describe introduced changes in more detail. :+1: 
You can check the example below :arrow_down: 
Screenshot from 2022-08-16 18-25-08


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**